content/en/docs/setup/windows/intro-windows-in-kubernetes.md

```diff
@@ -0,0 +1,400 @@
+* `securityContext.runAsGroup` -
+   not possible on Windows as there is no GID support
+* `securityContext.runAsNonRoot` -
+   this setting will prevent containers from running as `ContainerAdministrator`
+   which is the closest equivalent to a root user on Windows.
+* `securityContext.runAsUser` -
+   use [`runAsUserName`](/docs/tasks/configure-pod-container/configure-runasusername)
+   instead
+* `securityContext.seLinuxOptions` -
+   not possible on Windows as SELinux is Linux-specific
+* `terminationMessagePath` -
+   this has some limitations in that Windows doesn't support mapping single files. The
+   default value is `/dev/termination-log`, which does work because it does not
+   exist on Windows by default.
+
+##### Field compatibility for Pod specifications {#compatibility-v1-pod}
+
+The following list documents differences between how Pod specifications work between Windows and Linux:
+
+* `hostIPC` and `hostpid` - host namespace sharing is not possible on Windows
+* `hostNetwork` - There is no Windows OS support to share the host network
+* `dnsPolicy` - setting the Pod `dnsPolicy` to `ClusterFirstWithHostNet` is
+   not supported on Windows because host networking is not provided. Pods always
+   run with a container network.
+* `podSecurityContext` (see below)
+* `shareProcessNamespace` - this is a beta feature, and depends on Linux namespaces
+  which are not implemented on Windows. Windows cannot share process namespaces or
+  the container's root filesystem. Only the network can be shared.
+* `terminationGracePeriodSeconds` - this is not fully implemented in Docker on Windows,
+  see the [GitHub issue](https://github.com/moby/moby/issues/25982).
+  The behavior today is that the ENTRYPOINT process is sent CTRL_SHUTDOWN_EVENT,
+  then Windows waits 5 seconds by default, and finally shuts down
+  all processes using the normal Windows shutdown behavior. The 5
+  second default is actually in the Windows registry
+  [inside the container](https://github.com/moby/moby/issues/25982#issuecomment-426441183),
+  so it can be overridden when the container is built.
+* `volumeDevices` - this is a beta feature, and is not implemented on Windows.
+  Windows cannot attach raw block devices to pods.
+* `volumes`
+  * If you define an `emptyDir` volume, you cannot set its volume source to `memory`.
+* You cannot enable `mountPropagation` for volume mounts as this is not
+  supported on Windows.
+
+##### Field compatibility for Pod security context {#compatibility-v1-pod-spec-containers-securitycontext}
+
+None of the Pod [`securityContext`](/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context) fields work on Windows.
+
+### Node problem detector
+
+The node problem detector (see
+[Monitor Node Health](/docs/tasks/debug-application-cluster/monitor-node-health/))
+is not compatible with Windows.
+
+### Pause container
+
+In a Kubernetes Pod, an infrastructure or “pause” container is first created
```

Comment:
nit (not related to the refactor):
```suggestion
In a Kubernetes Pod, an infrastructure or _pause_ container is first created
```

I'll file an issue to add “pause container” to the glossary.